tests/jvm/src/test/scala/munit/TestLocalAsyncFixtureJVMSuite.scala

```diff
@@ -0,0 +1,33 @@
+package munit
+
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class TestLocalAsyncFixtureJVMSuite extends FunSuite {
```

Comment:
This suite was added for `JVM` and `JS` only because of the bug in native, see https://github.com/scala-native/scala-native/issues/1918